examples/touch/content/TabBarPage.qml

```diff
@@ -74,23 +74,23 @@ Page {
             frame: Item { }
             tab: Item {
                 implicitWidth: control.width/control.count
-                implicitHeight: 50
+                implicitHeight: size.dp(50)
```

Comment:
Not that I will be able to do anything about this, I should point out that having these bindings everywhere is going to hurt your creation performance badly. A binding (even a simple one like this) is massively more expensive than a simple property write.